docs/docs/README.md

```diff
@@ -9,12 +9,12 @@ slug: /
 
 Main features:
 
-- All-platform user sign-in / sign-up experience with SDK support (SPA, iOS and Android)
-- Passwordless (SMS / Email) support
-- Out-of-box social sign in support (GitHub, Google, WeChat, Alipay, etc.)
+- An OIDC-based identity service
+- All-platform user sign-in/up experience with dark mode and SDKs (Web, iOS, and Android)
+- Sign-in/up with dynamic SMS/Email passcode
```

Comment:
better align our term `register`: `Sign-in/up` → `Sign-in/register`?

Reply:
actually we do not use register for content or display. either create account or sign-up